llvm/test/TableGen/submulticlass-leteq.td

```diff
@@ -0,0 +1,21 @@
+// RUN: llvm-tblgen %s | FileCheck %s
+// XFAIL: vg_leak
+// CHECK:      def X0 { // C
+// CHECK-NEXT:   bit x = 1;
```

Comment:
This is `bit x = 0;` in the before case, yet `!cast<bits<8>>(-1)` is identical to `Val`.